cmake/stempyConfig.cmake.in

```diff
@@ -0,0 +1,10 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/stempyTargets.cmake")
+
+check_required_components(stempy)
+
+find_package(Python3 REQUIRED COMPONENTS Development)
+
+include_directories("${CMAKE_CURRENT_SOURCE_DIR}/stempy")
```

Comment:
I think this is where my issue is coming from. ${CMAKE_CURRENT_SOURCE_DIR} is not the same in the CMakeLists that imports. I removed both lines in a test branch [here](https://github.com/swelborn/stempy/commits/find-package-test/) and things still work. I believe the targets should hold this information, right?